### PR TITLE
completed e-commerce

### PR DESCRIPTION
**Pull request by coder-mezab**
i will completed my e-commerce project let me show what you guys think

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -192,3 +192,48 @@ nav ul li a:hover {
...
+}
+.png-img {
+  flex-basis: 20%;
+  min-width: 150px;
+}
+.png-img img {
+  width: 90%;
+  filter: grayscale(100%);
+}
+.png-img img:hover {
```
Comment (faran4engg) [inline css/style.css:214]: unique class name 😂

**Review by faran4engg — COMMENTED** [review]
@coder-mezab , Check the cart and menu.... Its not in center !! 
either make it in center, or align in the same line anywhere

Screen Shot 2021-09-27 at 12 49 28 PM

**Review by faran4engg — APPROVED** [review]
approving, merge it.
raise new PR for the issue I just submitted

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
Website looking good. For the first project..

**Pull request merged**